content/_getting-started/contracts-and-workspaces.md

```diff
@@ -110,19 +117,19 @@ client’s engineers, and production environment is customer-facing only.
 
 ## Limited Workspaces
 
-Workspaces can have the `limited` or `full` type. `Limited` is intended for platform trial period. Workspaces with `limited` type have certain restrictions for integration flows:
+The limited workspaces have certain restrictions for integration flows:
 
-- Integration flows in `limited` workspaces are restricted by work time. This means that they are automatically stopped after a certain time. This time is defined for each installation with an environment variable.
+*   Integration flows in the limited workspaces are restricted by work time. Meaning, the **platform automatically stops all active flows after 8 hours** in limited workspaces. You can restart the flows afterwards to continue your testing.
+*   If your integration flow contains messages in the processing queues, these messages will be removed when the time is up to stop the flow.
```

Comment:
* Drop the word `Meaning`
 * Consider adding the word `However,` to the start of the second bullet point and then pull it into the first paragraph.

Reply:
Yes, to first. However, using a `However` in the beginning of the bullet point would not be suitable. IMHO.

Reply:
Perhaps then create 3 bullet points with *You can restart the flows afterwards to continue your testing.* being it's own point.

Reply:
aha, good point - one moment will implement you suggestion

Reply:
Check the result please